### DISCUSSION TITLE
How to fix? i having this error

### DISCUSSION THREAD

**Discussion by alexandrucn** [FunkinCrew/Funkin#2079] [Compiling help]
source/PlayState.hx:150: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
C:/HaxeToolkit/haxe/lib/newgrounds/1,1,5/lib/Source/io/newgrounds/components/ScoreBoardComponent.hx:55: characters 24-44 : Warning : Std.is is deprecated. Use Std.isOfType instead.
C:/HaxeToolkit/haxe/lib/newgrounds/1,1,5/lib/Source/io/newgrounds/components/ScoreBoardComponent.hx:55: characters 49-66 : Warning : Std.is is deprecated. Use Std.isOfType instead.
C:\Users\acret\Desktop\Funkin-master\export\release\windows\obj>setlocal enabledelayedexpansion
Warning: Could not find Visual Studio 2017 VsDevCmd
Missing HXCPP_VARS
Error: Could not automatically setup MSVC

**Comment by Snirozu** [comment]
https://github.com/ninjamuffin99/Funkin#compiling-game
